src/pytest_inmanta_lsm/remote_service_instance_async.py

```diff
@@ -52,6 +53,7 @@ def get_service_instance_from_log(log: model.ServiceInstanceLog) -> model.Servic
             deployment_progress=None,
             service_identity_attribute_value=log.service_identity_attribute_value,
             referenced_by=None,
+            transfer_context=log.transfer_context,
```

Comment:
This is not required, for later versions of inmanta-lsm, which have these new attributes, the `log.to_service_instance` method is already define and should contain the compatibility logic